src/common/mod.rs

```diff
@@ -37,7 +38,7 @@ pub fn json_document_to_cbor(path: impl AsRef<Path>, protocol_version: Option<u3
 pub fn value_to_cbor(value: serde_json::Value, protocol_version: Option<u32>) -> Vec<u8> {
     let mut buffer: Vec<u8> = Vec::new();
     if let Some(protocol_version) = protocol_version {
-        buffer.write_u32::<BigEndian>(protocol_version);
+        buffer.write_u32::<BigEndian>(protocol_version).expect("write_u32 failed");
```

Comment:
This would cause a panic, killing the process if an invalid contract is passed, might be better to return a result?
I guess they are just for tests tho, so maybe it is fine.

Reply:
wouldn't it only fail if protocol_version isn't a u32?

Reply:
Yeah, is this just for test?
If it isn't then someone could send an invalid contract with the intent of causing a crash.